virtme_ng_init/src/main.rs

```diff
@@ -921,6 +921,9 @@ fn run_user_gui(tty_fd: libc::c_int) {
         // directly here so we may need extra permissions on the tty devices.
         utils::run_cmd("bash", &["-c", &format!("chown {} /dev/char/*", user)]);
 
+        // Clean up any previous X11 state.
+        utils::run_cmd("bash", &["-c", &"rm -f /tmp/.X11*/* /tmp/.X11-lock"]);
```

Comment:
detail: do you think there is an interest to combine this command with the previous one not to call `bash` twice in a row?

Reply:
We can probably save a tiny bit of time combining the two commands together, but I think it's negligible and since they're two completely different logical operations I'd prefer keep them separate for better readability.

Maybe if we need to add more commands to initialize xorg we may consider to create a `virtme/guest/virtme-xorg-script`, but at the moment I don't think it's worth it just for these two commands...

Reply:
Thank you for your reply! Fine for me to keep it like this!